test/net/sourceforge/kolmafia/preferences/PreferencesTest.java

```diff
@@ -559,6 +568,61 @@ public void exerciseIsPerUserGlobalProperty() {
     assertTrue(Preferences.isPerUserGlobalProperty("getBreakfast.PreferencesTestFakeUser"));
   }
 
+  public class timeinThread extends Thread {
+    public timeinThread(String s) {
+      super(s);
+    }
+
+    public void run() {
+      LoginManager.timein(CONCURRENT_USER_NAME);
```

Comment:
I was thinking this would be something more like:
```
while (not done running) {
  LoginManager.timein(CONCURRENT_USER_NAME);
}
```

(You may need to Thread.yield() execution periodically, not sure.)

Note that [Thread#stop is deprecated](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/doc-files/threadPrimitiveDeprecation.html).